src/main/java/nl/smallproject/www/techiteasy/controllers/RemoteControllersController.java

```diff
@@ -0,0 +1,61 @@
+package nl.smallproject.www.techiteasy.controllers;
+
+import jakarta.validation.Valid;
+import nl.smallproject.www.techiteasy.dtos.RemoteController.RemoteControllerInputDto;
+import nl.smallproject.www.techiteasy.dtos.RemoteController.RemoteControllerOutputDto;
+import nl.smallproject.www.techiteasy.dtos.RemoteController.RemoteControllerUpdateDto;
+import nl.smallproject.www.techiteasy.models.RemoteController;
+import nl.smallproject.www.techiteasy.services.RemoteControllerService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/remotecontroller")
+public class RemoteControllersController {
+    private final RemoteControllerService remoteControllerService;
+
+    public RemoteControllersController(RemoteControllerService remoteControllerService) {
+        this.remoteControllerService = remoteControllerService;
+    }
+
+    @RequestMapping(method = RequestMethod.GET)
+    public ResponseEntity<List<RemoteControllerOutputDto>> getAllRemoteControllers() {
+        List<RemoteControllerOutputDto> remoteControllers = remoteControllerService.getAllRemoteControllers();
+        return ResponseEntity.ok(remoteControllers);
+    }
+
+    @RequestMapping(value = "{id}", method = RequestMethod.GET)
```

Comment:
In deze endpoints zie ik niet terug dat wordt verwezen naar "/remotecontroller" (alleen {id}). Misschien dat dit ook werkt maar voor the sake of duidelijkheid zou ik "remotecontroller" hier ook in verwerken